YvesIraguha/TeaHouse
# Setup dev env 169663247

#### What does this PR do?
- set up the development environment
#### Description of Task to be completed?
- Set up babel, Eslint, and prettier
- Set up a testing environment
- Set up Travis 
#### How should this be manually tested?
- Clone the repo and checkout `setup-dev-env-169663247`
- Run `npm install && npm run test && npm run start` to start the project locally 
- You should see test running successfully, and server started successfully
#### Any background context you want to provide?
Coveralls, HoundCI, and CodeClimate on private repositories require paid plans, which I am not ready for yet. 
#### What are the relevant pivotal tracker stories?
# [169663247](https://www.pivotaltracker.com/story/show/169663247)
